server.py

```diff
@@ -0,0 +1,126 @@
+#python -m uvicorn server:app --reload
+
+import sqlite3
+import argparse  
+import random
+import string
+import sqlite_helpers
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.responses import RedirectResponse, HTMLResponse
+from datetime import datetime, timedelta
+
+app = FastAPI()
+
+#maybe create the table if it doesnt already exist
+sqlite_helpers.maybe_create_table("urldatabase.db")
+
+@app.post("/create_url")
+async def create_url(request: Request):
+
+    urljson = await request.json()
+    timestamp = datetime.now()
+
+    if "url" not in urljson:
+        raise HTTPException(status_code=400, detail="URL required.")
+
+    if "alias" not in urljson:
+        aliasVal = generate_alias()
```

Comment:
lets do a separate pr for `generate_alias`, this version of the server should expect `url` and `alias` to be sent

Reply:
done